### PR TITLE
apollo-starter template

### PR DESCRIPTION
**Pull request by ankitjena**
#17 Add apollo starter template using Apollo Server which uses typescript.

### CODE REVIEW

File: templates/apollo-starter-ts/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## apollo-starter-ts
+
+Starter template using graphback and apollo-server
+
+### Usage
+```
```
Comment (wtrocki) [inline templates/apollo-starter-ts/README.md]: No docker compose?

File: templates/apollo-starter-ts/docker-compose.yml [hunk 2]
```diff
@@ -12,8 +12,8 @@ services:
 
   # Mosca is a simple MQTT Broker
   # In OpenShift/Production we would use the Red Hat AMQ broker
-  mosca:
-    image: matteocollina/mosca
-    ports:
-      - "1883:1883" # MQTT
-      - "80:80" # web interface
+  # mosca:
```
Comment (wtrocki) [inline templates/apollo-starter-ts/docker-compose.yml]: We will need that for subscriptions. @matteocollina/mosca is the best MQTT broker for this job. Lets leave it

File: templates/apollo-starter-ts/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "apollo-template",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.ts",
```
Comment (wtrocki) [inline templates/apollo-starter-ts/package.json]: You ment dist/index.js

File: templates/apollo-starter-ts/src/db.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import knex = require('knex')
+
+export async function connect(options) {
```
Comment (wtrocki) [inline templates/apollo-starter-ts/src/db.ts]: Options missing type. Configure tslimt for templates

**Comment by wtrocki** [comment]
Apollo Express is an most popular one. Over the time we can also include hapi and fastify ones

**Comment by ankitjena** [comment]
> Apollo Express is an most popular one. Over the time we can also include hapi and fastify ones

Yes!

**Comment by wtrocki** [comment]
Good job.. With template ready we can think about the cli that will allow to pick template and generate further content. Great work man

**Pull request merged**